src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,7 +271,28 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
-
+        
+        public List<Object> getFormat() {
+        	HttpFuzzerReflectionDetectorStateHighlighter value = new HttpFuzzerReflectionDetectorStateHighlighter();

```

Comment:
This would be better if done in the other class, to not have the model depend on renderer classes.